images/win/scripts/Tests/Miniconda.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+Describe "Miniconda" {
+    It "Miniconda Environment variables is set. " {
+        Get-EnvironmentVariable "CONDA" | Should -Not -BeNullOrEmpty
+    }
+    
+    It "Miniconda $env:CONDA\<PathTest> " -TestCases @(
+        @{ PathTest = "python.exe" }
+        @{ PathTest = "Scripts\conda.exe" }
+    ) {
+        Join-Path ${env:CONDA} $PathTest | Should -Exist
```

Comment:
Just for consistency, let's use the same approach instead of different: `Get-EnvironmentVariable "CONDA"` and ` $env:CONDA`

Reply:
I have changed this.